comfyui_manager/prestartup_script.py

```diff
@@ -68,15 +65,17 @@ def is_import_failed_extension(name):
 comfy_base_path = os.environ.get('COMFYUI_FOLDERS_BASE_PATH')
 
 if comfy_path is None:
-    # legacy env var
-    comfy_path = os.environ.get('COMFYUI_PATH')
-
-if comfy_path is None:
-    comfy_path = os.path.abspath(os.path.dirname(sys.modules['__main__'].__file__))
+    try:
+        comfy_path = os.path.abspath(os.path.dirname(sys.modules['__main__'].__file__))
+        os.environ['COMFYUI_PATH'] = comfy_path
+    except:
+        print("[ComfyUI-Manager] environment variable 'COMFYUI_PATH' is not specified.")
+        exit(-1)
```

Comment:
Maybe it's worth considering throwing an exception here?

This file is imported, it is usually considered unnatural that a file that should be executed via import can call `exit` and terminate the entire program.